pepper-apis/src/main/resources/db-changes/schema/DDP-6520-singular-add-operator-id-to-age-up-candidate.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="ypavlov" id="20211006-add-operator_user_id-to-age_up_candidate">
+        <addColumn tableName="age_up_candidate" >
+            <column afterColumn="participant_user_id"
+                    name="operator_user_id"
+                    type="BIGINT(20)" >
+                <constraints references="user(user_id)" foreignKeyName="age_up_candidate_operator_user_fk" nullable="false" />
```

Comment:
since we are making this new column NOT NULL, we need to backfill this new column of existing rows with participant_user_id

Reply:
I put the query which sets the operator from governance if found or participant_user_id otherwise